ml-proto/src/parser.mly

```diff
@@ -51,7 +51,7 @@ type context =
   {funcs : space; globals : space; locals : space; labels : int VarMap.t}
 
 let empty () = {map = VarMap.empty; count = 0}
-let c0 () =
+let emptyContext () =
```

Comment:
Nit: empty_context (CamlCasing is only used for uppercase names in Ocaml, i.e., constructors and modules)